### PR TITLE
Add new UNIX permissions to Ubuntu install document

### LINKED ISSUE #5324: Update install documents with new UNIX permissions
If this is merged, the installation readmes (CentOS, "Install in depth", macOS) should be updated. 

Since at this time there are many open PRs dealing with these same files, it will be easier to make these changes once those PRs go through.

_Originally posted by @johnsaigle in https://github.com/aces/Loris/pull/5323#issuecomment-542811712_

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

UNIX permissions were changed in #5057. This updates the Ubuntu wiki to match what's in the main README.

- [x] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #5324 
* Resolves #6181 (new permissions will be included and documented)

### CODE REVIEW

**Comment by johnsaigle** [comment]
@christinerogers @driusan @ridz1208 This was always intended to be on the same branch as the new ReadTheDocs install documentation.

Since the install docs are now going to be included in the release, the changes from this PR should also be sent to 23.

Without these changes:
* users could run into permission issues, and 
* The install documentation don't match the "brief version" of the install in the main README.md.

**Comment by christinerogers** [comment]
@johnsaigle This was issued to the `master` branch. 
How about requesting that perhaps @AlexandraLivadas integrate these changes onto the 23 branch?    And then removing the `23 testing` label on this.

**Comment by johnsaigle** [comment]
> This was issued to the master branch.

Yes, as were the Ubuntu install instructions. As the install docs were exceptionally copied to the 23.0-release branch, the contents of this PR will also need to go there.

File: docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md [hunk 1]
```diff
@@ -138,6 +138,23 @@ Finally, change the _loris root_ directory owner to the _lorisadmin_ account
...
 ```
 
+## Creating the lorisadmin user
+
+```bash
+# Create lorisadmin user and group
+# Give lorisadmin `sudo` permission. This is required for the install process
+# in order to automatically generate Apache configuration files.
+# Sudo privileges can be revoked once the install is completed.
+sudo useradd -U -m -G sudo -s /bin/bash lorisadmin
```
Comment (ridz1208) [inline docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md:148]: @johnsaigle I know this subject has come up again and again... but im really not a fan of having lorisadmin be a sudoer !!

I understand we need sudo to do the initial install and configuration but cant we tell user to either : 
- run the following command with sudo credentials
or
- add lorisadmin to sudoers but then remove it later on

Comment (ridz1208) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md:148]: ALSO this PR conflicts with lines 108 to 140... can you check if the content is the same or different ?

Comment (johnsaigle) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md:148]: I'll delete what's above

Line 111-112 mentions that you can remove sudo after the install if that's what you mean

Comment (ridz1208) [reply docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md:121]: yeah... but I'd want it to be much more clear wording like SHOULD be revoked...

File: docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md [hunk 2]
```diff
@@ -111,32 +111,22 @@ This is required for the install process in order to automatically generate
...
-```
-
-Next, add Apache to the _lorisadmin_ group
+## Creating the lorisadmin user
 
 ```bash
+# Create lorisadmin user and group
+# Give lorisadmin `sudo` permission. This is required for the install process
+# in order to automatically generate Apache configuration files.
+# Sudo privileges can be revoked once the install is completed.
```
Comment (johnsaigle) [inline docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md:120]:
```suggestion
# 
# Sudo privileges should be revoked once the install is completed.
```

**Comment by ridz1208** [comment]
@driusan this is still adding the recommendation to have lorisadmin part of sudoers (which i dont like) but it mentions to remove it after the setup... it still beats switching to a root user and having all file created belong to root instead of lorisadmin

Unless you have a better solution, i'd say lets go with it

**Pull request merged**